File: tools/alarm/alarm_monitor/src/main/java/cn/linebead/snabp/alarm_monitor/service/dto/AlarmMessage.java
package cn.linebead.snabp.alarm_monitor.service.dto;

import com.alibaba.fastjson.annotation.JSONField;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;
import java.util.List;


@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class AlarmMessage implements Serializable {
    @JSONField( name = "type")
    public String type;

    @JSONField( name = "MessageType")
    public Integer MessageType;

    @JSONField(name = "RealAlarmVaryInfo")
    public List<RealAlarmVaryInfo> RealAlarmVaryInfo;
}

File: server/aspnet-core/modules/snabp-alarm/src/SnAbp.Alarm.Application.Contracts/Dtos/AlarmServer/AlarmMessage.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SnAbp.Alarm.Dtos
{
    public class AlarmMessage
    {
        public string type;
        public int MessageType;
        public List<RealAlarmVaryInfo> RealAlarmVaryInfo;
    }
}
